baseline_collision.py

```diff
@@ -190,24 +190,22 @@ def make_step_prediction(choice, confidence, heatmap_img=None):
         choice = 1
     if choice == 'implausible':
         choice = 0
-    if heatmap_img:
+    if heatmap_img is not None:
         xy_list = np.argwhere(heatmap_img >= 200)
-        xy_list = [{"x": x, "y": y} for x, y in xy_list]
+        xy_list = [{"x": int(x), "y": int(y)} for x, y in xy_list]
```

Comment:
The numpy int64 data type was causing an error in our serialization code, so I just converted them into normal ints.